api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -189,6 +189,17 @@ message AccessLogCommon {
   // To configure this value, users should configure
   // :ref:`custom_tags <envoy_v3_api_field_extensions.access_loggers.grpc.v3.CommonGrpcAccessLogConfig.custom_tags>`.
   map<string, string> custom_tags = 22;
+
+  // For HTTP: Total duration in milliseconds of the request from the start time to the last byte out.
+  // For TCP: Total duration in milliseconds of the downstream connection.
```

Comment:
Can you clarify what's the difference between this and `time_to_last_downstream_tx_byte`? The comment in stream_info.h for `requestComplete` explains.

Reply:
Thanks for pointing the stream_info.h file where to find the information. 

What would you suggest (see examples below)?
option 1: adding the explanation to the description
option 2: simply changing the description

_option 1: adding the explanation to the description_
```
  // For HTTP: Total duration in milliseconds of the request from the start time to the last byte out.
  // For TCP: Total duration in milliseconds of the downstream connection.
  // This is the total duration of the request (i.e., when the request's ActiveStream is destroyed)
  // and may be longer than ``time_to_last_downstream_tx_byte``.
```

 
_option 2: simply changing the description:_ 
```
  // The total duration in milliseconds of the request (i.e., when the request's ActiveStream is destroyed)
  // and may be longer than ``time_to_last_downstream_tx_byte``.
```